.github/workflows/on-release.yml

```diff
@@ -9,53 +9,58 @@ on:
-
     steps:
       # Providing the SSH PRIVATE of a user part of an admin group
       # is necessary to bypass PR checks
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
+          ref: 6.0.0.Final-jahia
           ssh-key: ${{ secrets.GH_SSH_PRIVATE_KEY_JAHIACI }}
 
+      - name: Set environment variables from parameters
+        shell: bash
+        run: |
+          echo "NEXUS_USERNAME=${{ secrets.NEXUS_USERNAME }}" >> $GITHUB_ENV
+          echo "NEXUS_PASSWORD=${{ secrets.NEXUS_PASSWORD }}" >> $GITHUB_ENV
+
       # Setting up the SSH agent to be able to commit back to the repository
       # https://github.com/webfactory/ssh-agent
       - uses: webfactory/ssh-agent@v0.7.0
         with:
           ssh-private-key: ${{ secrets.GH_SSH_PRIVATE_KEY_JAHIACI }}
 
-      - uses: jahia/jahia-modules-action/release@v2
-        name: Release Module
+      - name: Delete github tag ${{ github.event.release.tag_name }}
+        shell: bash
+        run: |
+          git config user.email 'jahia-ci@jahia.com'
+          git config user.name 'Jahia CI'
+          git fetch --all
+          git tag --delete ${{ github.event.release.tag_name }}
+          git push origin :refs/tags/${{ github.event.release.tag_name }}
+
+      - uses: actions/cache@v1
         with:
-          github_slug: Jahia/jcontent
-          primary_release_branch: master
-          release_id: ${{ github.event.release.id }}
-          release_version: ${{ github.event.release.tag_name }}
-          github_api_token: ${{ secrets.GH_API_TOKEN }}
-          nexus_username: ${{ secrets.NEXUS_USERNAME }}
-          nexus_password: ${{ secrets.NEXUS_PASSWORD }}
-
-      - uses: jahia/jahia-modules-action/release-publication@v2
-        name: Publish Module
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
         with:
-          module_id: jcontent
-          release_version: ${{ github.event.release.tag_name }}
-          nexus_username: ${{ secrets.NEXUS_USERNAME }}
-          nexus_password: ${{ secrets.NEXUS_PASSWORD }}
-
-      # Tmate only starts if any of the previous steps fails.
-      # Be careful since it also means that if a step fails the workflow will
-      # keep running until it reaches the timeout
-      - name: Setup tmate session
-        if: ${{ failure() }}
-        uses: mxschmitt/action-tmate@v3
-        timeout-minutes: 15
+          java-version: 1.8
+
+      - name: Release with Maven
+        run: |
+          if [[ ${{ github.event.release.tag_name }} == web* ]] ;
+          then
+            TAG_VERSION=$(echo "${{ github.event.release.tag_name }}") # e.g. 6.0.0.Final-jahia2
+            FINAL_RELEASE_VERSION=$(echo "$TAG_VERSION" | tr '_' '.')
+            REVISION=$(echo $FINAL_RELEASE_VERSION | tr -d 'jahia' | cut -d'-' -f2)
```

Comment:
Is it handling the case ther eis no number (I mean the first jahia often not contains -1)

Reply:
quelque chose comme : 
# Si REVISION est vide, assigner la valeur 0
if [ -z "$REVISION" ]; then
    REVISION=0
fi

Reply:
normally every version should contain a digit, but I'll add your suggestion just in case